lib/xhr.js

```diff
@@ -400,6 +411,9 @@ function enableXHROverH2(XMLHttpRequest, configuration) {
                     xhrInfo.get(self, 'password')
                 );
 
+                // Set native responseType
+                self._responseType = xhrInfo.get(self, 'responseType');
```

Comment:
@hthetiot  

Chrome complains if you try to set responseType = undefined to this line needs to be changed to 
```javascript
self._responseType = xhrInfo.get(self, 'responseType') || "";
```